src/pages/elements/app-icons/production.mdx

```diff
@@ -90,103 +90,29 @@ Stroke app icons feature both color and alpha channel gradients. This technique
   src="/videos/3_2-3_MakingAnIcon_Stroke_Step6.mp4"
 />
 
-## Exporting for product and web
+## Exporting SVGs for product and web
 
-App icons for product and web should be created for all background types. This ensures your icon will be accessible in any UI context.
+To be considered production-ready for product and web, confirm the following:
+
+- All artwork and artboards must be aligned to the pixel grid.
+- All strokes must be expanded (`Object > Expand`).
+- All possible shapes and paths should be combined.
+- Use only linear gradients. Do not use conical gradients, radial gradients, or gradients along stroke.
+- For stroke app icons, use the masking method described in the previous section. Do not use clipping masks.
+- If your icon **is not** a stroke app icon, you will need to place a 32 x 32px square in each of your artboards. This square should have no fill or stroke.
```

Comment:
This point isn't quite clear. It took me a second to realize what it meant.

If your icon **is not** a stroke app icon, _then_ you will need to place a 32 x 32px _transparent_ square _background_ on each ~of your~ artboard~s~. This square should have no fill or stroke _to insure that the icons maintains a squared size_.

Reply:
Im good with this 👍